src/components/Dialog/Dialog.tsx

```diff
@@ -14,6 +16,9 @@ type DialogProps = {
 export const Dialog: FC<DialogProps> = ({open, close, title, contentText, children, actions}) => {
   return (
     <MuiDialog open={open} onClose={close}>
+      <Box position="absolute" top={16} right={16}>
+        <CloseButton size={30} onClick={close} invertColors />
```

Comment:
len mi napadlo, ci by slo `Box component={CloseButton}` a mat to na jednom riadku 😄 ale not sure ani